common/core/src/commonMain/kotlin/com/trafi/maas/transit/ui/internal/NearbyTransitScheduleBadgesConstants.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("UNUSED_PARAMETER")
```

Comment:
What does this exactly ? 🤔

Reply:
Suppresses warnings from the Kotlin compiler that the class has members that are declared but not used.